### ISSUE TITLE
[occm] Build Release for 1.1.1 for Helm Chart

### ISSUE THREAD

**Issue by atsai1220** [kubernetes/cloud-provider-openstack#1596]
**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind bug

**What happened**:
Seems like occm 1.1.1 is merged into master but not yet released on the Helm chart.

**What you expected to happen**:
Expecting 1.1.1 to be available for consumption to resolve RBAC issues.

**How to reproduce it**:
```
❯ helm repo add cpo https://kubernetes.github.io/cloud-provider-openstack
❯ helm repo update
❯ helm search repo cpo
NAME                                    CHART VERSION   APP VERSION     DESCRIPTION                                  
cpo/openstack-cinder-csi                1.3.8           v1.21.0         Cinder CSI Chart for OpenStack               
cpo/openstack-cloud-controller-manager  1.0.1           v1.21.0         Openstack Cloud Controller Manager Helm Chart
cpo/openstack-manila-csi                1.1.2           v1.21.0         Manila CSI Chart for OpenStack               

```

**Anything else we need to know?**:
I may be under the incorrect assumption that 1.1.1 is available for consumption.

**Comment by ramineni** [comment]
@atsai1220 Currently, they are only released once the new branch is cut or on new release. You can expect it to be available by Aug first week once 1.22 is targeted to release.

**Comment by ramineni** [comment]
@atsai1220 New version of helm charts are released. https://github.com/kubernetes/cloud-provider-openstack/releases/tag/openstack-cloud-controller-manager-1.1.2 . please check.
